Cargo.toml

```diff
@@ -12,6 +12,10 @@ categories = ["containers", "networking", "podman"]
 exclude = ["/.cirrus.yml", "/.github/*", "/hack/*"]
 build = "build.rs"
 
+[package.metadata.vendor-filter]
+# This list is not exhaustive.
+platforms = ["x86_64-unknown-linux-gnu", "aarch64-unknown-linux-gnu", "powerpc64le-unknown-linux-gnu", "s390x-unknown-linux-gnu", "riscv64gc-unknown-linux-gnu"]
```

Comment:
I think we should try to include all linux platforms when possible. I do not know what packagers make use of this tarball and where netavark is packaged. I think this should continue to work for as much linux targets as possible. I don't want to cause regressions.

Reply:
Not having a platform in this list doesn't mean it won't build.  

It means that if there is a dependency that *only* exists on that platform, then the build would fail.  This is really unlikely - there's very very very few crates which e.g. *only* exist for say s390x-unknown-linux-gnu or so.

In the case that happens, it's not a big deal - the platform can be added.

Reply:
I still would like to have at least the musl targets included, I think alpine depends on this.

Reply:
OK done, though note that today that does not change the vendored set.  (It seems really unlikely for there to really ever be a crate which is only a dependency on musl targets)

Reply:
Fair enough, however I much rather have it in there just in case.